--- Makefile
+++ Makefile
@@ -9,12 +9,12 @@
 
 ##############################################################################
 ### compiler and path settings
-CC      = g++
-OPT     = -Wall -Wno-unknown-pragmas -O3 -g -mtune=native
-CFLAGS  =  
-LFLAGS  = -lgsl -lgslcblas 
-CPATHS  = -I. -I$(HOME)/local/include -I/opt/local/include -I/usr/local/include
-LPATHS  = -L$(HOME)/local/lib -L/opt/local/lib -L/usr/local/lib
+# CC      = g++
+# OPT     = -Wall -Wno-unknown-pragmas -O3 -g -mtune=native
+# CFLAGS  =  
+# LFLAGS  = -lgsl -lgslcblas 
+# CPATHS  = -I. -I$(HOME)/local/include -I/opt/local/include -I/usr/local/include
+# LPATHS  = -L$(HOME)/local/lib -L/opt/local/lib -L/usr/local/lib
 
 ##############################################################################
 # if you have FFTW 2.1.5 or 3.x with multi-thread support, you can enable the 
